deploy/pulse.conf

```diff
@@ -71,10 +80,11 @@ upstream staging_pulse {
 }
 
 server {
+  listen 443 ssl http2;
   server_name  staging.pulse.cio.gov;
 
-  ssl_certificate      /etc/nginx/ssl/keys/staging.pulse.cio.gov.chained.crt;
-  ssl_certificate_key  /etc/nginx/ssl/keys/staging.pulse.cio.gov.key;
+  ssl_certificate      /etc/letsencrypt/live/pulse.cio.gov/fullchain.pem;
+  ssl_certificate_key  /etc/letsencrypt/live/pulse.cio.gov/privkey.pem;
```

Comment:
This re-syncs the Pulse nginx config with what it is on the server, after re-updating the certificate (after the hard drive fault we suffered a month back) to use one from Let's Encrypt.